cueadmin/pyproject.toml

```diff
@@ -45,22 +45,22 @@ markers = [
 # --- Optional Test Dependencies ---
 [project.optional-dependencies]
 test = [
-    "pytest>=8.0.0",
-    "pytest-cov>=4.0.0",
-    "pytest-mock>=3.10.0",
-    "mock>=4.0.0",
-    "pyfakefs>=5.2.3"
+    "mock==2.0.0",
+    "pyfakefs==5.2.3",
+    "pytest",
+    "pytest-cov",
+    "pytest-mock",
```

Comment:
The PR description says: "Use flexible version constraints instead of pinned versions". These `==` constraints don't look flexible to me.